src/tls/Makefile

```diff
@@ -341,3 +364,10 @@ test13:
 ifeq ($(OS),Windows_NT)
     EXTRA_PATH = PATH="/usr/x86_64-w64-mingw32/sys-root/mingw/bin/:$(PATH)"
 endif
+
+# FFI support - calling from C into miTLS
+#depends on tls-gen
+tls-ffi: libmitls.dll
+
+libmitls.dll: mitls.cmxa #tls-gen
```

Comment:
Build performance - it lets me rebuild tls-ffi (re-linking the DLL) without having to wait for tls-gen to rebuild each time.  Not as useful now as it used to be, when the ffi.c was compiled by tls-ffi instead of tls-gen.  I'll uncomment the dependency.